checklists/alz_checklist.en.json

```diff
@@ -42,7 +42,7 @@
             "guid": "5d82e6df-6f61-42f2-82e2-3132d293be3d",
             "id": "A02.01",
             "severity": "Medium",
-            "link": "https://learn.microsoft.com/azure/cloud-adoption-framework/ready/landing-zone/design-area/azure-billing-microsoft-customer-agreement#design-recommendations"
```

Comment:
I think the updated link in the working branch is better for the partner access - the multi-tenant wouldn't be for partner access.

We could add it to A01.03 instead as the main link.

Reply:
@hgotar1 - This is the only one that I think needs adjustment from my side.